dashboard/app/models/concerns/levels/levels_within_levels.rb

```diff
@@ -73,6 +73,50 @@ def clone_child_levels(parent_level, new_suffix, editor_experiment: nil)
 
         return update_params
       end
+
+      # Create ParentLevelsChildLevel many-to-many relationships based on the
+      # contents of the specified levels' `contained_level_names` or `project_template_level_name` properties.
+      # Differentiated by level_kind (either contained or project template)
+      def setup_child_levels_for(levels, level_kind)
```

Comment:
I'd love to be more consistent about making the distinction between parent levels and child levels clear throughout this method; in particular, I think it would be helpful to always make sure we refer to a child level as `child_level` rather than just `level`; ie, 

```suggestion
      def setup_child_levels_for(levels, child_level_kind)
```